tests/test_base.py

```diff
@@ -52,6 +52,13 @@ def test_iconv_working(auto_container):
     )
 
 
+def test_group_nobody_working(auto_container):
+    """Test that nobody group is available in the container (bsc#1212118)."""
+    assert auto_container.connection.check_output(
+        "grep -qF 'nobody:' /etc/group"
+    )
```

Comment:
I'd suggest to check for the existence of the `nobody` group via some binary that queries the user db e.g.:

```suggestion
    assert auto_container.connection.check_output(
        "getent group nobody"
    )
```

Reply:
that's nicer